src/analysis/retail/mage/arcane/core/ArcaneBarrage.tsx

```diff
@@ -112,13 +120,31 @@ export default class ArcaneBarrage extends Analyzer {
 
     this.arcaneChargeTracker.clearCharges(event);
   }
+
+suggestions() {
```

Comment:
this is not the correct place to put this logic. if you want to run some code at the end of a fight, add an event listener for `Events.fightend` and run the code there.